yt_dlp/extractor/gopro.py

```diff
@@ -23,6 +23,8 @@ class GoProIE(InfoExtractor):
             'upload_date': '20210919',
             'uploader_id': 'fireydive30018',
             'duration': 396062,
+            'track': '',
+            'artist': '',
```

Comment:
The track is not supposed to be an empty string. Extractor's returning wrong value. You don't have to fix the code in this PR, but don't add wrong value to test.
```suggestion
```

Reply:
Split the fix into a separate commit.